util/dvsim/FlowCfg.py

```diff
@@ -83,10 +88,16 @@ def __init__(self, flow_cfg_file, proj_root, args):
         self.results_summary_server_html = ""
         self.results_summary_server_page = ""
 
-        # Full and summary results in md text.
+        # Full results in md test
```

Comment:
test? I thought text makes sense to me..

Reply:
wuups... typo